samcli/commands/init/interactive_init_flow.py

```diff
@@ -192,6 +197,7 @@ def _generate_from_use_case(
     [*] Test Function in the Cloud: sam sync --stack-name {{stack-name}} --watch
     """
     click.secho(next_commands_msg, fg="yellow")
+    runtime = get_custom_runtime_base_runtime(runtime) if is_custom_runtime(runtime) else runtime
```

Comment:
Can we define this value once above and re-use it avoid duplication of this line and line 173?